src/styles/dashboard.css

```diff
@@ -216,14 +216,15 @@ body {
 
 .netdata-chart-legend {
     position: absolute; /* within .netdata-container */
-    top: 0;
+    top: 26px;
+    bottom: 18px;
```

Comment:
👀

Reply:
@apostolidhs i think all the changes should be added also to `dashboard.slate.css`. Also, have you checked how it works with different legend position?
Apart from that, this file is meant to be included also in "static/custom dashboards" (html files). We haven't released react-dashboard as dashboard.js yet, but when we'll do it, it's better not to cause breaking changes in this case.